### ISSUE TITLE
How to input 𝄫(double flat) and 𝄪(double sharp) in REPL or vscode?

### ISSUE THREAD

**Issue by NeroBlackstone** [dpsanders/MusicTheory.jl#27]
I read [Unicode input documentation](https://docs.julialang.org/en/v1/manual/unicode-input/#Unicode-Input), and I realized there is no shortcut to input these two symbols.

So... How did you enter these two characters? If it's copy and paste, maybe I think we need an equivalent convenience.

**Comment by dpsanders** [comment]
Yes I think I copied and pasted. There is a way to add new latex abbreviations to the REPL, but I don't remember exactly how right now.

**Comment by NeroBlackstone** [comment]
I've added commonly used musical Unicode completions to Julia Repl, which are expected to be available in the next version.

These two symbols can be entered via `\flatflat` and `\doubledouble`.

I think when the next Julia is released, we can modify the `[compat]` entry and then close the issue.

**Comment by dpsanders** [comment]
Great thanks! I woukd have chosen the names `doubleflat` and `doublesharp`, which is what they are called at least in the UK.

**Comment by NeroBlackstone** [comment]
> Great thanks! I woukd have chosen the names `doubleflat` and `doublesharp`, which is what they are called at least in the UK. 

I chose this naming at first, but the maintainers of Julia recommend the current naming due to the discoverability of the symbols. 😺
